.nvmrc

```diff
@@ -1 +1 @@
-12.20.2
\ No newline at end of file
+14.19.0
```

Comment:
Use the latest patch version of the [@types/node](https://www.npmjs.com/package/@types/node) package with a matching major/minor version (currently 14.18.x)